realm/realm-library/src/androidTest/java/io/realm/RealmListTests.java

```diff
@@ -1019,4 +1022,213 @@ public void testRemoveAllFromRealmInvalidListShouldThrow() {
             assertEquals("This Realm instance has already been closed, making it unusable.", e.getMessage());
         }
     }
+
+    private interface ListModifier <T extends RealmObject> {
```

Comment:
Please add a small description for this interface. At a glance the name is not that telling.